api/HTMLCanvasElement.json

```diff
@@ -461,6 +357,212 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "options_alpha_parameter": {
+            "__compat": {
+              "description": "<code>options.alpha</code> parameter",
+              "spec_url": "https://www.khronos.org/registry/webgl/specs/latest/1.0/#WEBGLCONTEXTATTRIBUTES",
+              "support": {
+                "chrome": {
+                  "version_added": "32"
+                },
+                "chrome_android": {
+                  "version_added": "32"
+                },
+                "edge": {
+                  "version_added": "≤79"
+                },
+                "firefox": {
+                  "version_added": "30"
+                },
+                "firefox_android": {
+                  "version_added": "30"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "19"
+                },
+                "opera_android": {
+                  "version_added": "19"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "samsunginternet_android": {
+                  "version_added": "2.0"
+                },
+                "webview_android": {
+                  "version_added": "37"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "options_desynchronized_parameter": {
+            "__compat": {
+              "description": "<code>options.desynchronized</code> parameter",
+              "spec_url": "https://www.khronos.org/registry/webgl/specs/latest/1.0/#WEBGLCONTEXTATTRIBUTES",
+              "support": {
+                "chrome": [
+                  {
+                    "version_added": "81",
+                    "notes": "ChromeOS and Windows"
```

Comment:
Looks like https://bugs.chromium.org/p/chromium/issues/detail?id=945835 is the bug about enabling this.